_includes/v21.2/sql/locality-optimized-search-virtual-computed-columns.md

```diff
@@ -0,0 +1,5 @@
+CockroachDB's optimizer cannot plan {% if page.name == "cost-based-optimizer.md" %} locality-optimized searches {% else %} [locality-optimized searches](cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters) {% endif %} for queries that use [partitioned unique indexes](partitioning.html#partition-using-secondary-index) on [virtual computed columns](computed-columns.html#virtual-computed-columns).
```

Comment:
We should also mention here (or somewhere else) that as of 21.2 (which adds expression indexes) "locality optimized searches will not be planned on partitioned unique expression indexes".

Reply:
> FYI #10998 already added this known limitation for 21.1. The only difference in 21.2 is that expression indexes have been added, which have the same limitation

Ah ok - thanks for the heads up, did not know

> We should also mention here (or somewhere else) that as of 21.2 (which adds expression indexes) "locality optimized searches will not be planned on partitioned unique expression indexes".

Sounds good - added below the main paragraph: "Locality-optimized searches will also not be planned for queries that use partitioned unique [expression indexes](expression-indexes.html)."